installers/msi-language/StateDeploy/CustomAction.cs

```diff
@@ -267,6 +267,25 @@ private static ActionResult _installStateTool(Session session, out string stateT
                 return ActionResult.Failure;
             }
 
+            session.Log("Running prepare step...");
+            string prepareCmd = " _prepare";
+            string prepareOutput;
+            ActionResult prepareRunResult = ActiveState.Command.Run(session, stateToolPath, prepareCmd, out prepareOutput);
+            if (prepareRunResult.Equals(ActionResult.Failure))
+            {
+                //RollbarReport.Error(msg, session); ??
+                Record record = new Record();
+                var errorOutput = Command.FormatErrorOutput(prepareOutput);
+                record.FormatString = string.Format("state _prepare failed with error:\n{0}", errorOutput);
+
+                session.Message(InstallMessage.Error | (InstallMessage)MessageBoxButtons.OK, record);
+                return prepareRunResult;
+            }
+            else
+            {
+                session.Log(string.Format("Prepare Output: {0}", prepareOutput));
+            }
+
```

Comment:
@mdrohmann I'd like there to be logged info regardless of ActionResult value. Will this log the error output? Also, I was unsure about the need for rollbar reporting here, and very unsure about how to deal with the session.Message line. Any help is appreciated.